results.yml

```
@@ -800,27 +800,27 @@ translations:
 
 
   - key: table.percentages_table
-    t: Percentages
+    t: Porcentagens
   - key: table.rankings_table
     t: Rankings
```

Não sei se faz sentido, mas pode ser bom manter o padrão
```suggestion
    t: Classificações
```

---

Com certeza! Deve ter passado batido e esqueci de traduzir. Valeu